static/js/impala/suggestions.js

```diff
@@ -130,12 +130,14 @@ $.fn.searchSuggestions = function($results, processCallback, searchType) {
             $results.filter('.visible').removeClass('visible');
             return;
         }
+        var urlVal = encodeURIComponent($self.val());
```

Comment:
If you don't escape the HTML in the form value this would result in an XSS  vulnerability [1]. 

In this case you need to encode for a URL **and** also escape the value so that HTML can't be potentially be injected. 

To do that you just need to run `escape_()` on the URI-encode string. See also the previous comments and jsbin for an example.

[1] https://en.wikipedia.org/wiki/Cross-site_scripting